src/Api/include/Api/Orbit.h

```diff
@@ -316,14 +316,14 @@ extern ORBIT_EXPORT void* orbit_api_get_function_table_address_v0();
   orbit_api_v0 g_orbit_api_v0; \
   void* orbit_api_get_function_table_address_v0() { return &g_orbit_api_v0; }
 
-inline bool orbit_api_active() {
-  bool active = g_orbit_api_v0.active;
+inline bool orbit_api_enabled() {
+  bool enabled = g_orbit_api_v0.initialized && g_orbit_api_v0.enabled;
```

Comment:
Is it intended that you use `bool` here but `uint32_t` in `orbit_api_v0`? Are we assuming that C supports `bool` or not?

Reply:
Oh good observation. I'll fix this in another PR since it's not related to the change. Also, I have a C only version of OrbitTest locally that I need to submit. Thanks.

Reply:
Maybe create a bug (in case you haven't already).